### PR TITLE
getutxo endpoint and refactored addressinfo

### PR DESCRIPTION
**Pull request by Muneerah1612**
**What does this PR do?**
- Changed the address info endpoint logic
- Added import address endpoint
- added getutxo endpoint

A summary of what this PR is about

**List of tasks completed**

-   [x] Specify what tasks are completed in this PR
-   [ ] Specify what's left to be done

**How can this be tested?**

- Highlight how this PR can be tested

**Additional info**

Add more info if any. eg fixes #.

### CODE REVIEW

File: Backend/api/views/address_view.py [hunk 1]
```diff
@@ -1,54 +1,110 @@
...
+        )
         address.save()
         return Response(
-            {"status":status.HTTP_201_CREATED,"address":address.address_generated}
+            {"status": status.HTTP_201_CREATED, "address": address.address_generated}
         )
 
 
-class GetAddressInfo(APIView):
+class ImportAddress(APIView):
```
Comment (toshmanuel) [inline Backend/api/views/address_view.py]: Should this not be export address rather than import

File: Backend/api/views/address_view.py [hunk 2]
```diff
@@ -1,54 +1,110 @@
...
     """This view gets the address,
     script pubkey and redeem script, in the case where user want to import address to another wallet"""
-    permission_classes=(IsAuthenticated,)
-    def get(self,request,address):
-        addressinfo=Addresses.objects.filter(address_generated=address)
-        serializer=AddressSerializer(addressinfo, many=True)
+
+    # permission_classes=(IsAuthenticated,)
+    permission_classes = (AllowAny,)
```
Comment (toshmanuel) [inline Backend/api/views/address_view.py]: Why not Authenticated

File: Backend/api/views/address_view.py [hunk 3]
```diff
@@ -1,54 +1,110 @@
...
+
+    def get(self, request):
+        user_id = request.user.id
+        addressinfo = Addresses.objects.filter(user_id=user_id)
+        addr = [i.address_generated for i in addressinfo]
+        return Response({"addresses": addr})
+
+
+class GetAddressInfo(APIView):
+    permission_classes = (AllowAny,)
```
Comment (toshmanuel) [inline Backend/api/views/address_view.py]: Authenticate

**Review by toshmanuel — CHANGES_REQUESTED** [review]

File: Backend/api/views/address_view.py (hunk 1 above)
Comment (Muneerah1612) [reply Backend/api/views/address_view.py]: Oh, my bad. I will change it

File: Backend/api/views/address_view.py (hunk 2 above)
Comment (Muneerah1612) [reply Backend/api/views/address_view.py]: I forgot to change it back to IsAuthenticated, I could not test it on postman,so I tested it on the browser using the isallow permission class.

File: Backend/api/views/address_view.py (hunk 3 above)
Comment (Muneerah1612) [reply Backend/api/views/address_view.py]: Noted

**Review by j-kon — APPROVED** [review]
Well done mate

**Review by toshmanuel — APPROVED** [review]

**Pull request merged**